thavalon-webapp/src/styles/gameStyles/interactionStyles/missionManager.scss

```diff
@@ -0,0 +1,86 @@
+@import "../gameGlobals.scss"; 
+
+.mission-card-button-row {
+    width: 90%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+%mission-button {
+    @extend %game-button;
+    font-size: 0.8rem;
+    color: white;
+    padding: 2px 5px;
+}
+
+.mission-button-success {
+    @extend %mission-button;
+    background-color: $player-selected-color-primary;
+    &:hover {
+        background: rgba($player-selected-color-primary, $button-hover-opacity);
+    }
+}
+
+.mission-button-fail {
+    @extend %mission-button;
+    background-color: $button-red;
+    &:hover {
+        background: rgba($button-red, $button-hover-opacity);
+    }
+}
+
+.mission-button-reverse {
+    @extend %mission-button;
+    background-color: orange;
+    &:hover {
+        background: rgba(orange, $button-hover-opacity);
+    }
+}
+
+.mission-button-qb {
+    @extend %mission-button;
+    background-color: blue;
+    display: none;
+    &:hover {
+        background: rgba(blue, $button-hover-opacity);
+    }
+}
+
+.agravaine-declare-button {
+    @extend %game-button;
+    background-color: $button-red;
+    color: white;
+    &:hover {
+        background: rgba($button-red, $button-hover-opacity);
+    }
+}
+
+.mission-modal-close-button {
+    background-color: $button-red;
+    &:hover {
+        background: rgba($button-red, $button-hover-opacity);
+    }
+    position: relative;
+    float: right;
+    display: inline;
+    border: 1px solid;
+    border-radius: 10px;
+    padding: 3px 6px;
+    cursor: pointer; 
+    margin-top: 2px;
+    margin-bottom: 2px;
+    color: white;
+}
+
+.mission-header-passed {
+    color: green;
+}
+
+.mission-header-failed {
+    color: $button-red
```

Comment:
if we have a button-red constant should probably do one for green also for consistency? Even if we don't reuse green

Reply:
I had button-red since the red wasn't the same hex value as "red." I changed all of the colors to be button-<color> though so we can change them if needed in one spot.